hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/ratis/RatisHelper.java

```diff
@@ -206,6 +206,10 @@ static RaftClient newRaftClient(RpcType rpcType, RaftPeerId leader,
     GrpcConfigKeys.OutputStream.setOutstandingAppendsMax(properties,
```

Comment:
I think we can remove this setter. This is used by GrpcOutputStream in ratis which is not used by ozone.

Reply:
Done

Reply:
Done